### DISCUSSION TITLE
Skycultures Descriptions Upper Case Words

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2896] [Translations]
In stellarium-skycultures-descriptions there are for example phrase 518:

STAR (of) [1]GOD[1] Marduk, [2]GOD[2]N&#275;beru(Ford)

I ask if words written in upper case are special words which should not be  translate, because they are keywords. If you translate STAR to TÄHTI (in finnish) the Stellarium doesn't work. Is that so? If it is not, I continue translating them STAR, GOD, etc.

**Comment by gzotti** [comment]
Babylonian cuneiform texts seem to use "old" and "new" words, or words from different Mesopotamian languages. (Experts, please extend this, or check Wikipedia et al. It may also be a mix of special single glyphs (capitalized) and regular characters. Sorry, I forgot these details for now.) Those able to read them have developed a system of transliteration which mixes uppercase words with regular spelling to discern them. 
I don't see why TÄHTI should not work. But probably check with Finnish Babylonists about how to spell transliterated Babylonian terminology in Finnish.

**Reply by ilkant** [reply]
Thank you answering quickly. If Stellarium works, it is enough to me and I think that is enough to finnish users.
